### PR TITLE
[NEW] Admin

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
@RocketChat/ReactNative

Closes #ISSUE_NUMBER

ezgif com-video-to-gif

### CODE REVIEW

**Comment by diegolmello** [comment]
@pranay414 Excellent! Can you post a gif on PR description?

**Comment by pranavpandey1998official** [comment]
@diegolmello done. and you mentioned the wrong pranav :innocent:

**Comment by diegolmello** [comment]
> @diegolmello done. and you mentioned the wrong pranav 😇

😨Sorry about that!
Github always suggests me the other one and I have to pick your username.
It happened this time as well, but I messed up.
Again, great work 👊

File: app/views/SidebarView/index.js [hunk 1]
```diff
@@ -183,6 +206,15 @@ export default class Sidebar extends Component {
 					testID='sidebar-settings'
 					current={activeItemKey === 'SettingsStack'}
 				/>
+				{this.canSeeAdminPanel() ? (
+					<SidebarItem
+						text={I18n.t('Admin_Panel')}
+						left={<CustomIcon name='cog' size={20} color={COLOR_TEXT} />}
```
Comment (diegolmello) [inline app/views/SidebarView/index.js]: Use `shield-alt` icon

File: app/views/AdminPanelView/index.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ import styles from '../Styles';
 	authToken: state.login.user && state.login.user.token
 }))
 /** @extends React.Component */
-export default class AdminPanelView	 extends LoggedView {
+export default class AdminPanelView	extends LoggedView {
```
Comment (diegolmello) [inline app/views/AdminPanelView/index.js]: It's nitpicking, but it's so easy to fix that I need to request a change: there's a tab here.
Change for a normal space.

Comment (pranavpandey1998official) [reply app/views/AdminPanelView/index.js]: my vs code settings just changed and  have tab length equals to 1 space

**Comment by diegolmello** [comment]
@pranavpandey1998official Can you resolve conflicts?

**Pull request merged**